src/AudacityException.cpp

```diff
@@ -20,6 +20,21 @@ AudacityException::~AudacityException()
 {
 }
 
+void AudacityException::EnqueueAction(
```

Comment:
This PR introduces changes not directly related to the general purpose of this PR as i understand it. Also it introduces changes that will never be tested, however i do agree with neccessity of breaking complex action like "move and change" into separate steps

Reply:
Which changes do you think are unrelated?  I tried to explain reasons sufficiently in commit comments.

Reply:
moving CallAfter to EnqueueAction, and the description of this commit is indeed related to it's contents, but the commit itself does something that is not neccessary for this PR. And d45887740d738f22c80f77614a390b0ec41d5b65 also looks out of place (fix to a BasicUI)

Reply:
Understand that without commit d458877, this PR will fail to build on Windows, with link errors.

Reply:
CallAction is first moved out of an inline function, and into the .cpp file, so wx/app.h can be removed from the header file.  That requires adjustment of many other #include directives elsewhere.  So, I made a commit mostly about just those routine adjustments, separately, before presenting other changes in other commits.

Reply:
Ultimately wx/app.h must disappear from the .cpp as well as the .h file, so that I can claim that this new library is sufficiently toolkit-neutral.  I do that in another commit.

So yes, the total sequence of changes includes an intermediate state of the code transformation, which is never tested.  But I believe none the less that it is better to preserve a record of code transformations that are individually easier to review.